spec/models/observation_collection_spec.rb

```diff
@@ -1,9 +1,43 @@
 require 'spec_helper'
 
 describe ObservationCollection do
-  describe ".new" do
-    xit "" do
-      # add tests
+  describe ".new_from_array" do
+    it "returns new ObservationCollection" do
```

Comment:
hey, nice PR! :) like the design and the good testing. 

tiny thing: not sure this test here is needed. `#find_by`'s behavior is tested below, and so it's existence on instantiation as well. And if it behaves as an enumerable, whether it inherits from `Enumerable` or not is irrelevant.